src/_TextInputInnerVisualSlot.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {Box, Spinner} from '.'
+import {TextInputNonPassthroughProps} from './TextInput'
```

Comment:
do you think this is like a circular dependency?

Reply:
Possibly, but it's just a type. Do you have any alternative suggestions?